### ISSUE TITLE
Adding information about OSCORE support in Californium

### ISSUE THREAD

**Issue by rikard-sics** [coap-technology/coap-technology.github.io#27]
I am working at RISE in a project together with Ericsson on contributing support for OSCORE to Californium. Would it be possible to mention OSCORE somewhere including that Californium now has support for it? It is now included in the 2.0.x-branch of Californium.

**Comment by cabo** [comment]
Absolutely.  I probably will have time to do the necessary reorganizations in week 28.

**Comment by rikard-sics** [comment]
> Absolutely. I probably will have time to do the necessary reorganizations in week 28.

Thanks, that would be great.

**Comment by rikard-sics** [comment]
Just bringing this up again. It would be great if you could update the page with information about OSCORE when you have the time.

**Comment by cabo** [comment]
Absolutely, I just didn't manage to do this before my vacation started today...

**Comment by rikard-sics** [comment]
Perhaps information regarding OSCORE could be added in the section about the Californium implementation, to just mention that it supports OSCORE too? It is now included in the master Californium branch. I could create a pull request adding a short line about this if that's alright.

**Comment by cabo** [comment]
Maybe we need to overhaul the "specifications" page first.  This now lists about 1/3 of the relevant documents, see https://tools.ietf.org/html/draft-bormann-core-overview-01 for more completeness.
We could list OSCORE implementations there for the moment; we probably also need a long-term strategy for including this information in the implementation section.  Colorful badges sound like the right way to do this.

**Comment by rikard-sics** [comment]
> Maybe we need to overhaul the "specifications" page first. This now lists about 1/3 of the relevant documents, see https://tools.ietf.org/html/draft-bormann-core-overview-01 for more completeness.
Yes, starting by adding it there under a section about the OSCORE RFC can make sense.

I could make a PR to add information about the OSCORE RFC and the implementation in Californium to the specifications page if that's alright?

> We could list OSCORE implementations there for the moment; we probably also need a long-term strategy for including this information in the implementation section. Colorful badges sound like the right way to do this.

Yeah, badges showing what features (like OSCORE) a particular implementation supports could be a nice way.

**Comment by rikard-sics** [comment]
I made a PR adding a few lines describing OSCORE and the fact that Californium has support for it to the Specification page.
https://github.com/coap-technology/coap-technology.github.io/pull/32

**Comment by rikard-sics** [comment]
> I made a PR adding a few lines describing OSCORE and the fact that Californium has support for it to the Specification page.
> See #32

During the last meeting about OSCORE with Ericsson we discussed this further and realized it would be good to add a mention of OSCORE on the Overview page also. I have now updated the PR to add a short line about OSCORE to it also.
